### ISSUE TITLE
Spack recipe out there?

### ISSUE THREAD

**Issue by kurtsansom** [parthenon-hpc-lab/parthenon#590]
Is there a spack recipe for parthenon? there is not one in upstream spack, but maybe someone has made one?

**Comment by Yurlungur** [comment]
I don't believe there's a recipe yet. If you were interested in building one, I think that would be of interest!

**Comment by Yurlungur** [comment]
spack recipe exists but needs to be updated
